plugin/advisor/catalog/walk_through_test.go

```diff
@@ -23,6 +23,50 @@ var (
 
 func TestWalkThrough(t *testing.T) {
 	tests := []testData{
+		{
+			origin: &Database{
+				Name:   "test",
+				DbType: db.MySQL,
+			},
+			statement: `
+				CREATE TABLE t(a TEXT);
+				ALTER TABLE t ALTER COLUMN a SET DEFAULT '1';
+			`,
+			err: &WalkThroughError{
+				Type:    ErrorTypeInvalidColumnTypeForDefaultValue,
+				Content: "BLOB, TEXT, GEOMETRY or JSON column `a` can't have a default value",
+				Line:    3,
+			},
+		},
+		{
+			origin: &Database{
+				Name:   "test",
+				DbType: db.MySQL,
+			},
+			statement: `
+				CREATE TABLE t(a TEXT DEFAULT '1')
+			`,
+			err: &WalkThroughError{
+				Type:    ErrorTypeInvalidColumnTypeForDefaultValue,
+				Content: "BLOB, TEXT, GEOMETRY or JSON column `a` can't have a default value",
+				Line:    2,
+			},
+		},
+		{
+			origin: &Database{
+				Name:   "test",
+				DbType: db.MySQL,
+			},
+			statement: `
+				CREATE TABLE t(a INT NOT NULL);
+				ALTER TABLE t ALTER COLUMN a SET DEFAULT NULL;
+			`,
+			err: &WalkThroughError{
+				Type:    ErrorTypeSetNullDefaultForNotNullColumn,
+				Content: "Invalid default value for column `a`",
```

Comment:
nit: sentence should end with a period.

Reply:
But error contents in Bytebase have no period